docs/quality-assurance/bdd.md

```diff
@@ -0,0 +1,80 @@
+# Behavioural-Driven Design (BDD) at UKHO
+
+> "An approach that enhances communication between stakeholders and project team members by expressing product needs as concrete examples"
+
+This page contains information about BDD and how it can be used to aid how we develop software at UKHO.
+
+## What is BDD
+
+* A process to enable the entire team to better understand stories before they are played
+* BDD is about _collaboration_ - Product Owners (proxy for the customer), developers and testers communicating to gain an agreed understanding
+* Uses rules and examples to illustrate this understanding
+* When stories are played there is less ambiguity leading to better software development
+* BDD is NOT:
+  * Testing
+  * SpecFlow or Cucumber
+  * Just automated tests (tests are a downstream benefit)
+
+## Aims of BDD
+
+BDD aims to make the development of software more successful by:
+
+* Incorporating diverse perspectives when defining requirements
+* Creating a shared understanding in a delivery team of the required product changes
+* Reducing the amount of re-work required in the maintenance phase (by making the original change more complete)
+* Simplifying and shortening team Scrum events (refinement and planning)
+
```

Comment:
Might be worth adding something around helps ensure only code relevant to the behaviours is developed (MVP)

Reply:
Added